src/inmanta/ast/variables.py

```diff
@@ -242,7 +242,7 @@ def __init__(self, instance: Reference, attribute: LocatableString) -> None:
 
         # a reference to the instance
         self.instance = instance
-        self.location = range
+        self.location = attribute.location
```

Comment:
why is this changed?

Reply:
I think this is a mistake, I changed it.